### PR TITLE
Add translations for missed visits in the progress tab

### PR DESCRIPTION
**Pull request by govindkrjoshi**
**Story card:** [sc-8241](https://app.shortcut.com/simpledotorg/story/8241/include-missed-visits-in-progress-tab-cohort-reports)

## Because

We added a new string to the progress tab which was missing translations. 

## This addresses

This adds translations for the new string in other languages, except for `sid_ET`

### CODE REVIEW

**Review by PriyangaPKini — APPROVED** [review]
lgtm👍🏻

File: config/locales/api/am_ET.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-am-ET:
+am_ET:
```
Comment (harimohanraj89) [inline config/locales/api/am_ET.yml:1]: IIRC, this key should not change. It should remain hyphenated. Let's revert this diff. 

Transifex annoyingly makes this diff every time ([documented here](https://github.com/simpledotorg/simple-server/blob/master/doc/wiki/transifex-localization.md#3-check-and-fix-all-top-level-language-keys))

**Review by harimohanraj89 — CHANGES_REQUESTED** [review]
Blocking review.

**Review by harimohanraj89 — APPROVED** [review]
:shipit:

**Pull request merged**